src/main/java/io/github/thebusybiscuit/slimefun4/api/player/PlayerProfile.java

```diff
@@ -256,7 +242,9 @@ public void addWaypoint(@Nonnull Waypoint waypoint) {
             }
         }
 
-        if (waypoints.size() < 21) {
+        int maxWaypoints = settings.getInt("waypoints.max-allowed");
```

Comment:
not a fan of this. We should probably do it under teleporter instead as an item setting